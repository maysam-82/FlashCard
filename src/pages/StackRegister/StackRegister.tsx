import React from 'react';
import FormStack from '../../components/FormStack/FormStack';
import { RouteComponentProps } from 'react-router-dom';

const StackRegister: React.SFC<RouteComponentProps> = (props) => {
	return (
		<div className="container border rounded p-0 pb-2">
			<h4 className="text-center pt-2 pb-2 border-bottom  bg-primary rounded-top text-white">
				Create a New Stack
			</h4>
			<FormStack {...props} />
		</div>
	);
};

export default StackRegister;
