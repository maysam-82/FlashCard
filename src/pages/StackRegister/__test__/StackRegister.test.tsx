import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import StackRegister from '../StackRegister';
import { RouteComponentProps } from 'react-router-dom';

let props: RouteComponentProps;

describe('StackRegister component', () => {
	let component: ShallowWrapper;
	beforeEach(() => {
		component = shallow(<StackRegister {...props} />);
	});
	it('should render StackRegister', () => {
		expect(component).toMatchSnapshot();
	});
});
