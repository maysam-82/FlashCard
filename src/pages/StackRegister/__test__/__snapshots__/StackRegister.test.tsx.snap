// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StackRegister component should render StackRegister 1`] = `
<div
  className="container border rounded p-0 pb-2"
>
  <h4
    className="text-center pt-2 pb-2 border-bottom  bg-primary rounded-top text-white"
  >
    Create a New Stack
  </h4>
  <Connect(FormStack) />
</div>
`;
